workflows/prognostic_c48_run/runtime/transformers/fv3fit.py

```diff
@@ -84,7 +84,15 @@ def predict(self, inputs: State) -> State:
 
     def apply(self, prediction: State, state: State):
         if self.config.online:
-            state.update(prediction)
+            prediction_filled_nans = {}
+            for k, v in prediction.items():
+                try:
+                    existing_value = state[k]
+                except KeyError:
+                    prediction_filled_nans[k] = v
```

Comment:
This try/except is here because in the units tests there was a case of a predicting a quantity (`surface_precipitation_rate`) that was not in the state.